### DISCUSSION TITLE
Node graph UI ?

### DISCUSSION THREAD

**Discussion by nyue** [zauberzeug/nicegui#2938] [Q&A]
### Question

Is NiceGUI suitable for building Node Graph UI application like [such](https://reactflow.dev/) ?

If so, is there some examples I can use as a starting point ?

Kind regards

**Comment by falkoschindler** [comment]
Hi @nyue, you could certainly integrate a third-party library to create such a UI, but it can be challenging.
By the way, here is a similar discussion: https://github.com/zauberzeug/nicegui/discussions/1113
